src/platforms/common/data-management/event-grouping/grouping-enhancements.mdx

```diff
@@ -3,22 +3,17 @@ title: "Custom Grouping Enhancements"
 sidebar_order: 2
 ---
 
-If you use stack traces for grouping, the enhancements rules influence the data that feed into that algorithm.
-These rules can be configured on a per-project basis under _Project Settings > General Settings > Grouping Settings_.
+If you use stack traces for grouping, the enhancements rules influence the data that feed into that algorithm. These rules can be configured on a per-project basis under **Settings > [organization] > [project] > General: Grouping Settings**.
```

Comment:
I suggest using the exact path order that a user would have to follow in order to reach those settings.
```suggestion
If you use stack traces for grouping, the enhancements rules influence the data that feed into that algorithm. These rules can be configured on a per-project basis under **Settings > Projects > [project] > General Settings > Grouping Settings**.
```